src/sentry/static/sentry/app/views/organizationRepositories.jsx

```diff
@@ -78,6 +81,14 @@ class AddRepositoryLink extends PluginComponentBase {
     );
   }
 
+  parseGitHubRepo(repo) {
+    let re = /github.com\/(.+\/.+)/i;
```

Comment:
In situations where the repo doesn't exist (or they don't have permissions for it) we show an error saying GitHub returned a 404.